### PR TITLE
enable pypi releases through travis

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,21 +1,50 @@
 dist: xenial
 language: python
-
 python:
-  - "2.7"
-  - "3.5"
-  - "3.6"
-  - "3.7"
```
Comment (daenney) [inline .travis.yml]: Why are we removing 3.7 from the tests? 3.7 was release in June 2018 and is the current release of Python.

Comment (bastelfreak) [reply .travis.yml]: in the current travis config, 3.7 is marked as `allow_failures` and the tests don't pass. I can readd that if you want. I had a quick look but wasn't able to fix the CI failures. I removed the job to make it run faster.

Comment (daenney) [reply .travis.yml]: Mm, if the test don't pass lets leave that for now, but we should probably fix that before we go 1.0.

Comment (othalla) [reply .travis.yml]: Yep, we should fix 3.7 for next release and add it back without allow_failures

**Comment by othalla** [comment]
Looks fine

**Pull request merged**